airdrop/job/eligibility.py

```diff
@@ -35,6 +36,17 @@ def __populate_state(self):
             details["airdrop_id"] = window["airdrop_id"]
             details["rewards_processor"] = window["rewards_processor"]
             self._active_airdrop_window_map[window["airdrop_window_id"]] = details
+        logger.info("Active windows " + str(self._active_airdrop_window_map))
+
+        reward_windows = self._airdrop_db.execute("select aw.airdrop_id, ad.rewards_processor, aw.row_id as airdrop_window_id from airdrop_window aw, airdrop ad  " + \
```

Comment:
you can run this script on mainent ONLY after 11:00 AM , 25th jan today

Reply:
Any number of times between the registration end and claim start